itl/command-plugins.conf

```diff
@@ -1696,6 +1696,7 @@ object CheckCommand "by_ssh" {
 	}
 
 	vars.by_ssh_address = "$check_address$"
+	vars.by_ssh_port = "$ssh_port$"
```

Comment:
Hm, I still think this could be confusing if you don't expect a default value (and let the plugin handle it if no parameter is given). I understand that it might be convenient to just do so if the user defines `ssh_port`.